pipelines/HSC/DRP-RC2.yaml

```diff
@@ -92,14 +92,7 @@ tasks:
   objectTableCoreRefCatMatchWholeSkyPlot:
```

Comment:
you can remove all the configs expect "band" for matchedVisitCoreWholeSkyPlot and objectTableCoreWholeSkyPlot